test/FunctionalTests/Microsoft.OData.Core.Tests/TestCharArrayPool.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.OData.Buffers;
+
+namespace Microsoft.OData.Tests
+{
+    public class TestCharArrayPool : ICharArrayPool
```

Comment:
I remember I had a default implementation for ICharArrayPool. Don't I?

Reply:
kept the Testchararraypool as such to be reused later as the the testarraypool implementation was inside another class file.